subsys/bluetooth/host/audio/chan.c

```diff
@@ -2477,10 +2468,41 @@ int bt_audio_broadcast_sink_sync(struct bt_audio_broadcast_sink *sink,
 	}
 
 	sink->bis_count = num_bis;
+	sink->chan = chan;
 
 	return err;
 }
 
+int bt_audio_broadcast_sink_stop(struct bt_audio_broadcast_sink *sink)
+{
+	struct bt_audio_chan *chan;
+	int err;
+
+	CHECKIF(sink == NULL) {
+		BT_DBG("sink is NULL");
+		return -EINVAL;
+	}
+
+	chan = sink->chan;
+
+	if (chan->state != BT_AUDIO_CHAN_STREAMING ||
+	    chan->state != BT_AUDIO_CHAN_CONFIGURED) {
```

Comment:
Should be AND ("&&") - the state is going to be un-equal to at least one of them, so this test will always be true.

Reply:
Thanks!